test.js

```diff
@@ -1,47 +1,60 @@
+const { NetworkType } = require('symbol-sdk');
 const centiva = require('./lib/cjs/index');
+const SimpleWallet = centiva.SimpleWallet;
 
 run();
+runHD();
 
 async function run() {
     /**
      * create account
      */
-    // const acc = centiva.AccountUtil.generateAccount(152);
-    // console.log("Generated account's address: ", acc.address.address);
-    // console.log("Generated account's privKey: ", acc.privateKey.toString());
+    const acc = centiva.AccountUtil.generateAccount(152);
```

Comment:
If this is a general purpose dHealth library, it's better to use dHealth instead of centiva. It's ok to do this in a future PR. It's fine for now.